src/pages/conference/timeline/_schedule.tsx

```diff
@@ -0,0 +1,176 @@
+import styled from 'styled-components'
+import ArrowRight from './images/downwards_right.svg'
+import ArrowLeft from './images/downwards_left.svg'
+import { Header } from 'components/elements'
+import device from 'themes/device'
+
+type ScheduleProps = {
+    time?: string | number
+    id?: string
+    name?: string
+    topic?: string
+}
+
+type DataProps = {
+    timings: ScheduleProps[]
+    free_time: string | number
+    speakers: ScheduleProps[]
+}
+
+type ScheduleConferenceProps = {
+    id?: string
+    title?: string
+    date?: string
+    period?: string
+    data?: DataProps[]
+}
+
+type ConferenceProps = {
+    item: ScheduleConferenceProps
+    key: string
+}
+
+const MainContainer = styled.div`
+    display: flex;
+    justify-content: center;
+`
+
+const Container = styled.div`
+    padding-bottom: 80px;
+`
+
+const HeaderTitle = styled.div`
+    padding-bottom: 80px;
+`
+
+const WrapperContainer = styled.div`
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    width: 100%;
+    max-width: 995px;
+`
+
+const ImageContainer = styled.div`
+    @media ${device.tablet} {
+        margin-left: 40px;
+    }
+`
+
+const TimeWrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+
+    h2:last-child {
+        margin-bottom: 0;
+    }
+
+    @media ${device.tablet} {
+        margin-left: 40px;
+    }
+`
+
+const ContentWrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+
+    h2:last-child {
+        margin-bottom: 0;
+    }
+`
+
+const ScheduleConference = ({ item }: ConferenceProps) => {
+    const { id, date, data, period, title } = item
+
+    return (
+        <>
+            <HeaderTitle key={id}>
+                <Header as="div" weight="400" type="subtitle-1" align="center" pt="16px">
+                    {title}
+                </Header>
+                <Header as="div" weight="400" type="subtitle-1" align="center" pt="16px">
+                    {date}
+                </Header>
+                <Header as="div" weight="400" type="subtitle-1" align="center" pt="16px">
+                    {period}
+                </Header>
+            </HeaderTitle>
+
+            <Container key={id}>
+                {data.map((item, index) => {
+                    return (
+                        <>
+                            <MainContainer>
+                                <WrapperContainer>
+                                    <TimeWrapper>
+                                        {item.timings.map((item) => {
+                                            return (
+                                                <Header
+                                                    as="div"
+                                                    type="heading-3"
+                                                    align="center"
+                                                    weight="700"
```

Comment:
We don't need this due the default properties

Reply:
removed the weight property